.github/workflows/LINUX_BUILD_TEST.yml

```diff
@@ -153,13 +154,583 @@ jobs:
+          BIN_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/binaries/CloverLeaf/openmp/cloverleaf_gcc8.3.0_armv8.4
+          PASS_STRING: "This test is considered PASSED"
+
+      - if: ${{ inputs.SIMENG-MODE == 'Release' }}
+        name: CloverLeaf openmp gcc9.3.0 armv8.4
+        uses: ./.github/actions/simeng_benchmarks
+        with:
+          RUN_DIR: $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/CloverLeaf
+          BIN_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/binaries/CloverLeaf/openmp/cloverleaf_gcc9.3.0_armv8.4
+          PASS_STRING: "This test is considered PASSED"
+
+      - if: ${{ inputs.SIMENG-MODE == 'Release' }}
+        name: CloverLeaf openmp gcc10.3.0 armv8.4
+        uses: ./.github/actions/simeng_benchmarks
+        with:
+          RUN_DIR: $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/CloverLeaf
+          BIN_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/binaries/CloverLeaf/openmp/cloverleaf_gcc10.3.0_armv8.4
+          PASS_STRING: "This test is considered PASSED"
+
+      - if: ${{ inputs.SIMENG-MODE == 'Release' }}
+        name: CloverLeaf openmp armclang20 armv8.4
+        uses: ./.github/actions/simeng_benchmarks
+        with:
+          RUN_DIR: $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/CloverLeaf
+          BIN_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/binaries/CloverLeaf/openmp/cloverleaf_armclang20_armv8.4
+          PASS_STRING: "This test is considered PASSED"
+
+      - if: ${{ inputs.SIMENG-MODE == 'Release' }}
+        name: miniBUDE openmp gcc8.3.0 armv8.4
+        uses: ./.github/actions/simeng_benchmarks
+        with:
+          DATA_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/miniBUDE/bm1
+          BIN_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/binaries/miniBUDE/openmp/minibude_gcc8.3.0_armv8.4
+          PASS_STRING: "Largest difference was 0.000%."
+
+      - if: ${{ inputs.SIMENG-MODE == 'Release' }}
+        name: miniBUDE openmp gcc9.3.0 armv8.4
+        uses: ./.github/actions/simeng_benchmarks
+        with:
+          DATA_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/miniBUDE/bm1
+          BIN_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/binaries/miniBUDE/openmp/minibude_gcc9.3.0_armv8.4
+          PASS_STRING: "Largest difference was 0.000%."
+
+      - if: ${{ inputs.SIMENG-MODE == 'Release' }}
+        name: miniBUDE openmp gcc10.3.0 armv8.4
+        uses: ./.github/actions/simeng_benchmarks
+        with:
+          DATA_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/miniBUDE/bm1
+          BIN_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/binaries/miniBUDE/openmp/minibude_gcc10.3.0_armv8.4
+          PASS_STRING: "Largest difference was 0.000%."
+
+      - if: ${{ inputs.SIMENG-MODE == 'Release' }}
+        name: miniBUDE openmp armclang20 armv8.4
+        uses: ./.github/actions/simeng_benchmarks
+        with:
+          DATA_PATH: $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/miniBUDE/bm1
```

Comment:
Following on from previous comments, here you could have something along the lines of:  `PARAM_STRING: "-n n64 -i 1 --deck " + $GITHUB_WORKSPACE/simeng-benchmarks/Data_Files/miniBUDE/bm1` . Not sure of the syntax, but hopefully this illustrates the point